<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="left_frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="title_box">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>188</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="message_board">
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="center_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QGridLayout" name="image_viewer"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="right_frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0" colspan="2">
         <widget class="QTreeView" name="workflow_tree"/>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="edit_op_button">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTreeView" name="op_tree"/>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="rm_op_button">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="apply_workflow_button">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="add_op_button">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>workflow_tree</zorder>
       <zorder>edit_op_button</zorder>
       <zorder>op_tree</zorder>
       <zorder>edit_op_button</zorder>
       <zorder>add_op_button</zorder>
       <zorder>apply_workflow_button</zorder>
       <zorder>rm_op_button</zorder>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>left_frame</zorder>
   <zorder>right_frame</zorder>
   <zorder>center_frame</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCalibrate">
   <property name="text">
    <string>HDF5 Builder</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionImage_1">
   <property name="text">
    <string>Image 1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
